app/components/error_list/error_list.js

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {
+    Dimensions,
+    StyleSheet,
+    View,
+    TouchableOpacity
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import FormattedText from 'app/components/formatted_text';
+import GeneralError from './general_error';
+import Config from 'assets/config.json';
+
+const {width: deviceWidth} = Dimensions.get('window');
+
+const style = StyleSheet.create({
```

Comment:
Never mind. It looks like we don't theme the error bar on the webapp. We can consider it in the future, but it's not needed now.